src/main/java/automation/examples/site/pages/home/HomePageImpl.java

```diff
@@ -1,21 +1,36 @@
 package automation.examples.site.pages.home;
 
+import automation.examples.site.components.competition.CompetitionFragment;
+import automation.examples.site.components.login.LoginFragment;
+import com.codeborne.selenide.ElementsCollection;
 import com.codeborne.selenide.SelenideElement;
 import org.slf4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import static com.codeborne.selenide.Condition.exactText;
 import static com.codeborne.selenide.Condition.visible;
 import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
 import static com.codeborne.selenide.Selenide.open;
 
-
 @Component
 public class HomePageImpl implements HomePage {
 
+    @Autowired
+    private CompetitionFragment competitionFragment;
+
     @Autowired
     private Logger logger;
 
+    @Autowired
+    private LoginFragment loginFragment;
+
+    @Override
+    public LoginFragment getLoginFragment() {
+        return loginFragment;
+    }
+
     @Override
     public void openPage() {
         logger.debug("Logger works!");
```

Comment:
This is useless log message. Paraphrase pls.

Reply:
This logger will be used in future projects. And added like template by @yuz123